api/data_explorer/test/test_facets_controller.py

```diff
@@ -35,6 +35,9 @@ def create_app(self):
         app = super(TestFacetsController, self).create_app()
         app.config.update({
             'INDEX_NAME': 'index_name',
+            'UI_FACETS': {
+                'Region': None
```

Comment:
Could we set to something so we can test it works?

Reply:
Done